File: summer/WebContent/js/system/windows/RequestFlags.js
define(["dojo/_base/declare"], function(declare){

    var RequestFlags=declare("RequestFlags", null,{ 
    });	
    
    RequestFlags.FullyResolved=0;
	RequestFlags.AnimationBaseValue=1;
	RequestFlags.CoercionBaseValue=2;
	RequestFlags.DeferredReferences = 4;
	RequestFlags.SkipDefault = 8;
	RequestFlags.RawEntry = 16;

    
    return RequestFlags;
    
});

File: org.summer.view.widget/src/org/summer/view/widget/RequestFlags.java
package org.summer.view.widget;
public enum RequestFlags 
{ 
    FullyResolved ,//= 0x00,
    AnimationBaseValue ,//= 0x01, 
    CoercionBaseValue ,//= 0x02,
    DeferredReferences ,//= 0x04,
    SkipDefault ,//= 0x08,
    RawEntry ,//= 0x10, 
}